css/styles.css

```diff
@@ -183,4 +193,13 @@ body {
   flex-shrink: none;
 }
 
+/*Hidden List with Null  */
+.yesList {
```

Comment:
you don't need both classes and names are not really self explained. 
I can propose to use something like: 
```js
.hidden {
  visibility: hidden;
}
```